packages/cf-component-checkbox/src/CheckboxGroup.js

```diff
@@ -1,8 +1,19 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import Checkbox from './Checkbox';
+import { applyTheme } from 'cf-style-container';
 import includes from 'lodash/includes';
 
+import CheckboxUnstyled from './Checkbox';
+import CheckboxTheme from './CheckboxTheme';
+
+const Checkbox = applyTheme(CheckboxUnstyled, CheckboxTheme, theme => ({
```

Comment:
How about adding a bool prop `group` to the `cf-component-checkbox/checkbox`? Then you can do `marginTop: group ? ...` there. 

It's less code and you can reuse the original Checkbox component instead of creating this new one.